### ISSUE TITLE
Cannot build for node-webkit

### ISSUE THREAD

**Issue by songziming** [mapbox/node-sqlite3#395]
I've been trying to compile node-sqlite3 for node-webkit for a whole day, and still not success.

My OS is ubuntu 14.04 64bit, and the command I use is:

```
sudo npm install sqlite3 --build-from-source --runtime=node-webkit --target_arch=x64 --target="0.11.5"
```

If I compile using clang, it'll say something like:

```
  TOUCH Release/obj.target/deps/action_before_build.stamp
  CC(target) Release/obj.target/sqlite3/gen/sqlite-autoconf-3080701/sqlite3.o
clang: error: unknown argument: '-fno-tree-vrp'
clang: error: unknown argument: '-fno-tree-sink'
make: *** [Release/obj.target/sqlite3/gen/sqlite-autoconf-3080701/sqlite3.o] Error 1
```

If I use gcc toolchain, it'll report an error like:

```
 CXX(target) Release/obj.target/node_sqlite3/src/database.o
../src/database.cc: In static member function ‘static void node_sqlite3::Database::New(const v8::FunctionCallbackInfo<v8::Value>&)’:
../src/database.cc:134:71: error: no matching function for call to ‘v8::Object::Set(v8::Local<v8::String>, v8::Local<v8::String>, v8::PropertyAttribute)’
     args.This()->Set(NanNew("filename"), args[0]->ToString(), ReadOnly);
                                                                       ^
../src/database.cc:134:71: note: candidates are:
In file included from /home/szm/.nw-gyp/0.11.5/src/node.h:76:0,
                 from ../src/database.cc:2:
/home/szm/.nw-gyp/0.11.5/deps/v8/include/v8.h:2116:8: note: bool v8::Object::Set(v8::Handle<v8::Value>, v8::Handle<v8::Value>)
   bool Set(Handle<Value> key, Handle<Value> value);
```

**Comment by Mithgol** [comment]
That's because the fix from #369 has not yet made it to a new npm package of `sqlite3`.

**Comment by springmeyer** [comment]
suppor for node-webkit v0.11.x is totally broken with no plan to fix in sight: #404
